### DISCUSSION TITLE
I don't think you need to URL-encode VM names for macOS shortcuts

### DISCUSSION THREAD

**Discussion by mjgardner** [utmapp/UTM#4225] [Show and tell]
@conath: Contra [wiki](https://github.com/utmapp/UTM/wiki/URL-Scheme-%E2%80%93-Automation-for-UTM-app-&-VMs#url-parameter-encoding), I'm able to use a human-readable name with spaces in my macOS shortcut to start a VM. It looks like the URL action handles the encoding for you.

<img width="751" alt="image" src="https://user-images.githubusercontent.com/133483/180559584-365355cb-e17f-48c3-b230-238307c442f6.png">

**Comment by conath** [comment]
Thanks @mjgardner for letting us know! I have updated the Wiki page to mention the automatic URL encoding in Shortcuts. For completeness: when automating UTM in other ways, including in Terminal, the parameters do need to be URL-encoded. (`open utm://start…`).
